packages/dashboard/src/components/queryEditor/iotSiteWiseQueryEditor/footer/disableAdd.spec.tsx

```diff
@@ -123,4 +124,90 @@ describe(disableAdd, () => {
       )
     ).toBeFalsy();
   });
+
+  describe('for unmodeled data', () => {
```

Comment:
Should you also be testing xy-plot here like in the modeled tests?

Reply:
sure, was trying to keep the file small, no other particular reason